corehq/apps/app_manager/views/utils.py

```diff
@@ -415,6 +415,37 @@ def form_has_submissions(domain, app_id, xmlns):
     return FormES().domain(domain).app([app_id]).xmlns([xmlns]).count() != 0
 
 
+def get_new_multimedia_between_builds(domain, target_build_id, source_build_id, build_profile_id=None):
+    source_build = get_app_cached(domain, source_build_id)
+    target_build = get_app_cached(domain, target_build_id)
+    assert source_build.copy_of, _("Size calculation available only for builds")
+    assert target_build.copy_of, _("Size calculation available only for builds")
```

Comment:
Where would these AssertionErrors get caught? Does the user ever see this message?

It would be better to make this more formal:
```
if not (source_build.copy_of and target_build.copy_of):
    raise ValueError(_("Size calculation available only for builds"))
```

Reply:
This is a way to just make sure that our expectation is enforced that this method is called only for builds. The user should actually never see this.